client-js/README.md

```diff
@@ -0,0 +1,202 @@
+# CEP-78 JavaScript Client Tutorial
+
+This tutorial outlines the usage of the JavaScript client available for the CEP-78 Enhanced NFT Standard.
+
+Further information on the CEP-78 Enhanced NFT Standard can be found [here](https://github.com/casper-ecosystem/cep-78-enhanced-nft).
+
+## Installing Dependencies
+
+Dependencies can be installed using the following command in the `client-js` directory:
+
+```js
+npm i
+```
```

Comment:
I think after release it will be looking different, because the package will be published through the npm. 
So then you there should be `npm i casper-cep78-js-client` rather then cloning this repo :)